schemas/vertices/features.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "A component of a DNA sequence, such as a CDS, mRNA, etc.",
+  "required": [
+    "_key",
+    "type",
+    "location"
+  ],
+  "properties": {
+    "_key": {
+      "type": "string",
+      "title": "Content hash",
+      "description": "Hash of the DNA sequence for this feature."
+    },
+    "protein_translation": {
+      "type": "string",
+      "title": "Protein translation",
+      "description": "Longest coded protein (representative protein for splice variants)"
+    },
+    "protein_translation_length": {
+      "type": "integer",
+      "description": "Length of protein_translation"
+    },
+    "protein_hash": {
+      "type": "string",
+      "title": "Protein content hash",
+      "description": "Hash of the protein sequence that this feature encodes."
+    },
+    "note": {
+      "type": "string",
+      "description": "Free-text description of this feature"
+    },
+    "functions": {
+      "type": "array",
+      "title": "Gene functions",
+      "items": {"type": "string"}
+    },
+    "functional_descriptions": {
+      "type": "array",
+      "title": "Gene function descriptions",
+      "items": {"type": "string"}
+    },
+    "type": {
+      "type": "string",
+      "examples": ["Gene", "ncRNA", "repeat", "CDS", "mRNA"]
```

Comment:
I switched this to the "Type" as determined by the types in the different arrays rather than which array the feature occurs in. It's a little more helpful I thing because non_coding_features is a real grab bag of types.

Reply:
that sounds a lot better